src/App/Platforms/Android/Services/DeviceActionService.cs

```diff
@@ -553,6 +558,98 @@ public long GetActiveTime()
             // ref: https://developer.android.com/reference/android/os/SystemClock#elapsedRealtime()
             return SystemClock.ElapsedRealtime();
         }
+
+        public async Task ReturnToPasskeyAfterUnlock()
+        {
+            var activity = (MainActivity)Microsoft.Maui.ApplicationModel.Platform.CurrentActivity;
```

Comment:
⚠️ Cast it using `as MainActivity` so it doesn't throw an exception and the next `if null` works as expected.